src/client/ui/dialogs/FileDialog.scss

```diff
@@ -1,9 +1,3 @@
-:local(.fileDialogModalContainer) {
-  display: flex;
-  flex-direction: column;
-  flex: 1;
-}
```

Comment:
FileDialog should probably set a min-width and min-height styles. It gets really small at times, like when you navigate to an empty folder.